request-example.go

```diff
@@ -209,6 +214,7 @@ func newMemFile(name string, isdir bool) *memFile {
 		name:    name,
 		modtime: time.Now(),
 		isdir:   isdir,
+		content: []byte(""),
```

Comment:
Is it important that this be set to an empty `[]byte{}` and not use a `nil`?

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
That was a quick fix I made early in testing to move on and is now unnecessary. I'll remove it.